opentracing_shim/src/main/java/io/opentelemetry/opentracingshim/SpanContextShim.java

```diff
@@ -33,8 +32,15 @@
   private final io.opentelemetry.trace.SpanContext context;
   private final DistributedContext distContext;
 
+  public SpanContextShim(SpanShim spanShim) {
+    this(
+        spanShim.telemetryInfo(),
+        spanShim.getSpan().getContext(),
+        spanShim.contextManager().contextBuilder().build());
+  }
+
   public SpanContextShim(TelemetryInfo telemetryInfo, io.opentelemetry.trace.SpanContext context) {
-    this(telemetryInfo, context, EmptyDistributedContext.getInstance());
+    this(telemetryInfo, context, telemetryInfo.contextManager().contextBuilder().build());
```

Comment:
this method chaining is pretty ugly. Can we make a convenience method on `TelemetryInfo` that does this work?

Reply:
Actually I had been thinking of adding a 'getDefault()' or `getEmpty()` method or similar to `DistributedContextManager` (so we don't have to create a new builder every time we need an *empty* instance).

Reply:
That being said, for now I guess we could have a `DistributedContext createEmptyContext()` or similar, to avoid this one ;)